### ISSUE TITLE
Guidelines for contributing a lesson

### ISSUE THREAD

**Issue by olexandr-konovalov** [LibraryCarpentry/lc-second-website#26]
> Develop the initial content on GitHub using the Carpentries' set up instructions for new lessons. New lessons are built from clones of this repo.

That repository has a template for the Software Carpentry - we need to check whether it's explained somewhere how to change the logo.

> Propose your lesson for the incubator by making a pull request (PR) against the Library Carpentry website, adding a link to your lesson repository as an alpha lesson.

Where to specify the alpha status is not clear - perhaps it will be clarified after #23 ?

**Comment by libcce** [comment]
Updated information is at https://librarycarpentry.org/lessons/. Please have a look and if you think further clarification is needed, please submit a followup issue here https://github.com/LibraryCarpentry/librarycarpentry.github.io describing what additional information/instructions are needed. Thank you so much!
